go.mod

```diff
@@ -32,22 +32,24 @@ require (
 	github.com/onsi/ginkgo v1.10.2 // indirect
 	github.com/opencontainers/go-digest v1.0.0-rc1.0.20180430190053-c9281466c8b2
 	github.com/opencontainers/runc v1.0.0-rc5.0.20180920170208-00dc70017d22 // indirect
-	github.com/openshift/api v0.0.0-20200109182645-c3cf38ec5571
-	github.com/openshift/client-go v0.0.0-20200109173103-2763c6378941
-	github.com/openshift/library-go v0.0.0-20200110080616-97e47564adf6
+	github.com/openshift/api v0.0.0-20200210091934-a0e53e94816b
+	github.com/openshift/client-go v0.0.0-20200116152001-92a2713fa240
+	github.com/openshift/library-go v0.0.0-20200309152735-d18a5ca14d82
 	github.com/pborman/uuid v1.2.0
 	github.com/prometheus/client_golang v1.1.0
 	github.com/prometheus/procfs v0.0.8 // indirect
 	github.com/satori/go.uuid v1.2.1-0.20180103174451-36e9d2ebbde5 // indirect
 	github.com/sirupsen/logrus v1.4.2
+	golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543 // indirect
 	google.golang.org/cloud v0.0.0-20151119220103-975617b05ea8 // indirect
 	gopkg.in/yaml.v2 v2.2.7
-	k8s.io/api v0.17.0
-	k8s.io/apimachinery v0.17.0
-	k8s.io/client-go v0.17.0
+	k8s.io/api v0.17.2
+	k8s.io/apimachinery v0.17.2
+	k8s.io/client-go v0.17.1
```

Comment:
@ricardomaraschini why the skew between k8s.io/api and k8s.io/client-go?

Reply:
well, that was not on purpose but I chosen to left it anyways :-) do you want me to revert?

Reply:
It has to do with the library-go update:

https://github.com/openshift/library-go/blob/master/go.mod